scripts/jenkins/ardana/ansible/roles/crowbar_register/tasks/main.yml

```diff
@@ -4,6 +4,22 @@
   set_fact:
     crowbar_node_name: "d{{ hostvars[inventory_hostname]['ansible_%s' | format(admin_interface)].macaddress | replace(':', '-') }}.{{ cloud_fqdn }}"
 
+- name: Create crowbar node list properly sorted
+  set_fact:
+    crowbar_node_list: "{{ crowbar_node_list | default('crowbar.' + cloud_fqdn) }} {{ hostvars[item]['crowbar_node_name'] }}"
+  loop: "{{ groups['cloud_virt_hosts'] }}"
+  delegate_to: "{{ ardana_env }}"
+  run_once: true
+
+- name: Fill in custom crowbar node list in mkcloud.conf
+  lineinfile:
+    dest: "{{ admin_mkcloud_config_file }}"
+    regexp: "^export custom_crowbar_node_list"
+    line: "export custom_crowbar_node_list='{{ crowbar_node_list }}'"
```

Comment:
I would put "order" somewhere in the name of this variable to better describe it's use.

Reply:
done - changed to `custom_crowbar_node_order`